### PR TITLE
Fix some Coverity issues

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: tests/mocks/Matchers.cpp [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 #include "Matchers.h"
 
+#include <cctype>;
```
Comment (nomis52) [inline tests/mocks/Matchers.cpp]: No ; at the end of the line.

Comment (peternewman) [inline tests/mocks/Matchers.cpp]: Done

**Comment by peternewman** [comment]
And I've just done it again! Sorry @nomis52

**Comment by nomis52** [comment]
LGTM

**Pull request merged**